wiki/Ranking_Criteria/osu!mania/es.md

```diff
@@ -39,9 +34,9 @@ Este conjunto de **criterios de clasificación de osu!mania** establece [reglas
 - **Chord**: Dos o más notas presionadas al mismo tiempo.
 - **Chordstream (Jumpstream, Handstream, Quadstream)**: Streams que mezclan diferentes tipos de chords.
 - **Grace**: Dos o más notas presionadas en sucesión rápida, generalmente usando snaps de 1/6 o más.
-- **Hand**: Tres notas presionadas al mismo tiempo. Ocasionalmente referida como una "triple".
+- **Hand**: Tres notas presionadas al mismo tiempo. Ocasionalmente referida como una «triple».
```

Comment:
Is there a need of using guillemets here instead of quotation marks? I'd understand if this was for example another language like Korean which uses CJK brackets rather than quotation but I personally do not see the need of using guillemets here, I opt for quotation marks for simplification sake considering the audience.

This goes for all of them.

Reply:
Better ask agatemosu for that, but that seems to be the way they've been standardized to.

It's more "correct" to use these in Spanish however, as explained _(Not)_ briefly in point D in https://www.rae.es/dpd/comillas.

Reply:
Resolved using the cited source above as the reason on why we're using these quotation marks instead.